src/sqlancer/ComparatorHelper.java

```diff
@@ -84,35 +84,42 @@ public static List<String> getResultSetFirstColumnAsString(String queryString, E
         return resultSet;
     }
 
-    public static void assumeResultSetsAreEqual(List<String> resultSet, List<String> secondResultSet,
-            String originalQueryString, List<String> combinedString, SQLGlobalState<?, ?> state) {
-        if (resultSet.size() != secondResultSet.size()) {
-            String queryFormatString = "-- %s;\n-- cardinality: %d";
-            String firstQueryString = String.format(queryFormatString, originalQueryString, resultSet.size());
-            String secondQueryString = String.format(queryFormatString,
-                    combinedString.stream().collect(Collectors.joining(";")), secondResultSet.size());
-            state.getState().getLocalState().log(String.format("%s\n%s", firstQueryString, secondQueryString));
-            String assertionMessage = String.format("the size of the result sets mismatch (%d and %d)!\n%s\n%s",
-                    resultSet.size(), secondResultSet.size(), firstQueryString, secondQueryString);
+    public static void assumeResultSetsAreEqual(List<String> firstResultSet, List<String> secondResultSet,
+            String originalQuery, List<String> combinedQuery, SQLGlobalState<?, ?> state) {
+        if (firstResultSet.size() != secondResultSet.size()) {
+            String queryFormatString = "-- %s;" + System.lineSeparator() + "-- cardinality: %d" + System.lineSeparator()
+                    + System.lineSeparator() + System.lineSeparator();
+            String firstQueryString = String.format(queryFormatString, originalQuery, firstResultSet.size());
+            String combinedQueryString = String.join(";", combinedQuery);
+            String secondQueryString = String.format(queryFormatString, combinedQueryString, secondResultSet.size());
+            state.getState().getLocalState()
+                    .log(String.format("%s" + System.lineSeparator() + "%s", firstQueryString, secondQueryString));
+            String assertionMessage = String.format(
+                    "the size of the result sets mismatch (%d and %d)!" + System.lineSeparator() + "firstQuery: %s"
+                            + System.lineSeparator() + "secondQuery: %s" + System.lineSeparator()
+                            + System.lineSeparator() + System.lineSeparator(),
+                    firstResultSet.size(), secondResultSet.size(), firstQueryString, secondQueryString);
             throw new AssertionError(assertionMessage);
         }
 
-        Set<String> firstHashSet = new HashSet<>(resultSet);
+        Set<String> firstHashSet = new HashSet<>(firstResultSet);
         Set<String> secondHashSet = new HashSet<>(secondResultSet);
 
         if (!firstHashSet.equals(secondHashSet)) {
             Set<String> firstResultSetMisses = new HashSet<>(firstHashSet);
```

Comment:
here, what firstResultSetMisses should be (secondHashSet-firstHashSet)?, e.g. 
```java
Set<String> firstResultSetMisses = new HashSet<>(secondHashSet);
firstResultSetMisses.removeAll(firstHashSet);
```

Reply:
Sounds good. But it is a name conversion issue, which is not related to the current issue according to your description. Can you create a separate PR to submit all your modifications about name conversions?